f2/src/locales/en.json

```diff
@@ -49,15 +49,14 @@
   "impactPage.nextButton": "Next: Contact information",
   "impactPage.summary": "Tell us more about how it impacted you.",
   "impactPage.title": "Impact caused by the scam",
-  "landingPage.intro": "Scams can happen to anyone. They’re not always easy to recognize and new ones are invented every day.",
+  "landingPage.intro": "Your report helps the RCMP learn aoubt these crimes and helps police across Canada track down cybercriminals.",
   "landingPage.nextButton": "Report now",
-  "landingPage.required0": "Before you start, gather what you can about the scam:",
-  "landingPage.required1": "Dates when it took place",
-  "landingPage.required2": "Descriptions of what happened",
-  "landingPage.required3": "Details on where it came from",
+  "landingPage.required0": "Before you start, gather any evidence about the crime or scam:",
+  "landingPage.required1": "Description of what happened",
+  "landingPage.required2": "Dates when teh events took place",
```

Comment:
Typo

Reply:
Thank you for catching the mistake. I fixed it.